### PR TITLE
GTALL-205_School_capacity

### PR DESCRIPTION
**Pull request by silviadimova**
ticket URL

### CODE REVIEW

File: app/src/main/java/bubi/com/logintest/school/School.java [hunk 1]
```diff
@@ -13,6 +13,10 @@
     private List<Teacher> teacherList = new ArrayList<>();
     private Headteacher headteacher;
 
+    public void setCapacity(int capacity) {
+        this.capacity = capacity;
```
Comment (stoichoandreev) [inline app/src/main/java/bubi/com/logintest/school/School.java:17]: Excellent usage of **this**

File: app/src/test/java/bubi/com/logintest/MyProgramm.java [hunk 2]
```diff
@@ -41,15 +41,26 @@ public void testSchoolNameSaved(){
...
+        school.setCapacity(150);
 
         addNewStudentIfNotEnough(school.getStudentNumber());
         addNewTeacherIfZero(school.getTeacherNumber());
         printHeadteacherNameIfExisting(school.getHeadteacher().getName());
-
         printSchoolFoundationYear();
+        printSchoolCapacity(school.getCapacity());
 
+    }public void printSchoolCapacity(int capacity){
```
Comment (stoichoandreev) [inline app/src/test/java/bubi/com/logintest/MyProgramm.java]: This method should start on new line not next to the above method's {

File: app/src/test/java/bubi/com/logintest/MyProgramm.java [hunk 3]
```diff
@@ -41,15 +41,26 @@ public void testSchoolNameSaved(){
...
+        if(capacity<150){
```
Comment (stoichoandreev) [inline app/src/test/java/bubi/com/logintest/MyProgramm.java]: Please add spaces between **capacity < 150**, thanks

**Review by stoichoandreev — CHANGES_REQUESTED** [review]

File: app/src/main/java/bubi/com/logintest/school/School.java (hunk 1 above)
Comment (silviadimova) [reply app/src/main/java/bubi/com/logintest/school/School.java:17]: Java is the smart, not me :-)))

Comment (stoichoandreev) [reply app/src/main/java/bubi/com/logintest/school/School.java:17]: Remind me to explain you how **this** actually works

Comment (silviadimova) [reply app/src/main/java/bubi/com/logintest/school/School.java:17]: ok

**Comment by stoichoandreev** [comment]
@silviadimova please fix the requested code changes so we can close your PR

**Comment by silviadimova** [comment]
Az им дадох адд и commit на промените. Не помня пак ли пуш трябваше

    On Tuesday, November 12, 2019, 02:25:24 PM GMT, Stoycho Andreev <notifications@github.com> wrote:  
 
 
@silviadimova please fix the requested code changes so we can close your PR

**Comment by stoichoandreev** [comment]
Yep, if you don't push, your changes are only saved locally, nothing else.
In order to share them with the rest , you need to push them to the Github, just push to the same feature branch

**Comment by silviadimova** [comment]
готово
    On Tuesday, November 12, 2019, 02:31:38 PM GMT, Stoycho Andreev <notifications@github.com> wrote:  
 
 
Yep, if you don't push, your changes are only saved locally, nothing else.
In order to share them with the rest , you need to push them to the Github, just push to the same feature branch

**Comment by stoichoandreev** [comment]
I am afraid that is not готово :))))). Maybe try again, because your changes are not in this PR

**Review by stoichoandreev — APPROVED** [review]
LGTM

**Pull request merged**